src/ios/Calendar.m

```diff
@@ -392,8 +392,9 @@ - (NSMutableArray*) eventsToDataArray: (NSArray*)matchingEvents {
     if (event.location != nil) {
       [entry setObject:event.location forKey:@"location"];
     }
-    if (event.notes != nil) {
-      [entry setObject:event.notes forKey:@"message"];
+    if (event.notes != nil && event.URL != nil) {
```

Comment:
I think right the problem is if any of the values are nil it won't show any.

I think it should be something like:
```
if (event.notes != nil || event.url != nil) {
      NSString *notes = event.notes ?: @""
      NSString *url = event.URL ?: @""
      NSString *notesWithURL = [NSString stringWithFormat:@"%@ %@", notes, url];
      [entry setObject:notesWithURL.trimmingCharacters(in: .whitespacesAndNewlines) forKey:@"message"];
}	    
```
My only issue with this is that if there are no notes and only url it will have leading space. 
Is there trim method in objective-C?
i only found this one:
string.trimmingCharacters(in: .whitespacesAndNewlines) But not sure how it works=)

Reply:
Will address these changes :)